.github/CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 /avm/ptn/aca-lza/hosting-environment/ @Azure/avm-ptn-acalza-hostingenvironment-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/ptn/ai-platform/baseline/ @Azure/avm-ptn-aiplatform-baseline-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/ptn/authorization/policy-assignment/ @Azure/avm-ptn-authorization-policyassignment-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/ptn/authorization/policy-exemption/ @Azure/avm-ptn-authorization-policy-exemption-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
Either the team is not created or not yet approved in the hierachy ([ref](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only))

Reply:
Got it, must be that it hasn't been approved yet then as it exists [here](https://github.com/orgs/Azure/teams/avm-ptn-authorization-policy-exemption-module-owners-bicep).

Reply:
Hey @oZakari, please don't resolve a comment unless it's a code suggestion 😉 Makes re-reviews a bit harder.
In this case this must remain a blocking comment until the group was approved. It could lead to issues if we merge the module before.

Reply:
Bumping as tests are still failing for group approval. @matebarabas

Reply:
I believe this is all resolved, but will let you close @AlexanderSehr